u-boot/include/soc/qca_soc_common.h

```diff
@@ -1344,7 +1344,7 @@
 		#define QCA_PLL_CPU_PLL_DITHER_NFRAC_MIN_H_SHIFT	5
 		#define QCA_PLL_CPU_PLL_DITHER_NFRAC_MIN_H_MASK		BITS(QCA_PLL_CPU_PLL_DITHER_NFRAC_MIN_H_SHIFT, 13)
 		#define QCA_PLL_CPU_PLL_DITHER_NFRAC_STEP_SHIFT		18
-		#define QCA_PLL_CPU_PLL_DITHER_NFRAC_STEP_MASK		BITS(QCA_PLL_CPU_PLL_DITHER_NFRAC_MIN_SHIFT, 6)
+		#define QCA_PLL_CPU_PLL_DITHER_NFRAC_STEP_MASK		BITS(QCA_PLL_CPU_PLL_DITHER_NFRAC_STEP_SHIFT, 6)
```

Comment:
same fix applies below for non qca956x chips

Reply:
Thank you, bad copy and paste.

Hopefully, qca955x are not using NFRAC_STEP.